Formula/antidote.rb

```diff
@@ -0,0 +1,34 @@
+class Antidote < Formula
+  desc "Plugin manager for zsh, inspired by antigen and antibody"
+  homepage "https://getantidote.github.io/"
+  url "https://github.com/mattmc3/antidote/archive/refs/tags/v1.6.4.tar.gz"
+  sha256 "336ea89aab33d07d5d2eaa14af2f015d7da657837f64d3e76fa5c54645210950"
+  license "MIT"
+
+  uses_from_macos "zsh"
+
+  def install
+    pkgshare.install "antidote.zsh"
+    pkgshare.install "functions"
+    rm_f Pathname.glob("man/*.md")
+    share.install "man"
```

Comment:
Actually, why not just
```suggestion
    man.install "man/man1"
```
?

Reply:
On an early incarnation of this, I was getting the markdown files intermingled in the man page locations. `man.install "man/man1"` seems to handle this correctly and didn't pollute `/usr/local/share/man`. Nice catch.